deploy/chart/values-edge.yaml

```diff
@@ -1,11 +1,33 @@
 deploy:
   ecs: false
 
-csi:
-  disk:
-    enabled: false
-  local:
-    enabled: true
-  oss:
-    controller:
-      enabled: false
+nodepools:
+  default:
+    pluginname: csi-plugin
+    nodeSelectorTerms:
+      - matchExpressions:
+        - key: type
+          operator: NotIn
+          values:
+            - virtual-kubelet
+        - key: alibabacloud.com/is-edge-worker
+          operator: In
+          values:
+            - "false"
+  edge:
+    pluginname: csi-plugin-edge
+    csi:
+      disk:
+        enabled: false
+      local:
+        enabled: true
+      oss:
+        controller:
+          enabled: false
```

Comment:
The same here, move to top level. Configuring controller in nodepool makes no sense.

Reply:
okay